index.css

```diff
@@ -1,8 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap');
```

Comment:
- [ ] Please I highly suggest sticky to the [Figma Design] as mentioned by the previous reviewer. 😄 

**REQUIRED FIGMA DESIGN**

![Screenshot (73)](https://user-images.githubusercontent.com/78104544/227654149-3fff537b-6014-44ea-abe1-8039d6ebaa42.png)


_**YOUR DESIGN**_

![Screenshot (99)](https://user-images.githubusercontent.com/78104544/227654213-25762a6b-5aa7-4ef9-881f-73d5ec209db7.png)